docs/04-java-client/01-client-overview.md

```diff
@@ -6,36 +6,73 @@ permalink: /docs/java-client/client-overview
+Client to the Cadence service used to start and query workflows by external processes
 
-### com.uber.cadence.workflow
+### [com.uber.cadence.workflow](https://javadoc.io/doc/com.uber.cadence/cadence-client/latest/com/uber/cadence/workflow/Workflow.html)
 APIs to implement workflows.
 
-### com.uber.cadence.worker
+### [com.uber.cadence.worker](https://javadoc.io/doc/com.uber.cadence/cadence-client/latest/com/uber/cadence/worker/package-summary.html)
 APIs to configure and start workers.
 
-### com.uber.cadence.testing
+### [com.uber.cadence.testing](https://javadoc.io/doc/com.uber.cadence/cadence-client/latest/com/uber/cadence/testing/package-summary.html)
 APIs to write unit tests for workflows.
 
-## [Samples](https://github.com/cadence-workflow/cadence-java-samples/tree/master/src/main/java/com/uber/cadence/samples)
-### com.uber.cadence.samples.hello
+## Cadence Java Samples
+### [com.uber.cadence.samples.hello](https://github.com/cadence-workflow/cadence-java-samples/tree/master/src/main/java/com/uber/cadence/samples/hello)
 Samples of how to use the basic feature: activity, local activity, ChildWorkflow, Query, etc.
 This is the most important package you need to start with.
-### com.uber.cadence.samples.bookingsaga
+### [com.uber.cadence.samples.bookingsaga](https://github.com/cadence-workflow/cadence-java-samples/tree/master/src/main/java/com/uber/cadence/samples/bookingsaga)
 An end-to-end example to write workflow using SAGA APIs.
-### com.uber.cadence.samples.fileprocessing
+### [com.uber.cadence.samples.fileprocessing](https://github.com/cadence-workflow/cadence-java-samples/tree/master/src/main/java/com/uber/cadence/samples/fileprocessing)
 An end-to-end example to write workflows to download a file, zips it, and uploads it to a destination.
 
  An important requirement for such a workflow is that while a first activity can run
 on any host, the second and third must run on the same host as the first one. This is achieved
  through use of a host specific task list. The first activity returns the name of the host
   specific task list and all other activities are dispatched using the stub that is configured with
  it. This assumes that FileProcessingWorker has a worker running on the same task list.
+
+
+## Differences between Java and Golang Clients
+
+While many features are already supported in the Cadence Golang client, some features are still missing in the Java client. The Cadence development team aims for feature parity between the two clients, with new features typically being implemented in Golang first.
+
+### Feature Comparison
+
+| Feature                                                   | Go Client      | Java Client   | Priority |
+|-----------------------------------------------------------|----------------|---------------|----------|
+| **Shadow tests**                                           | Supported      | No            | P0       |
```

Comment:
I think we technically have this, we just never onboarded anyone to it within Uber.

Reply:
Great! https://github.com/cadence-workflow/cadence-java-client/blob/b0f0a7edf453a62a23f6a814e3a0c0ffb543d484/src/main/java/com/uber/cadence/testing/WorkflowShadower.java#L38